File: Python/BaseballGame.py
def baseballGame(operations):
    stack = []

    for operation in operations:
        if operation == 'C':
            stack.pop()
        elif operation == 'D':
            stack.append(2*stack[len(stack)-1])
        elif operation == '+':
            stack.append(stack[len(stack)-1] + stack[len(stack)-1])
        else:
            stack.append(int(operation))

    result = 0
    for item in stack:
        result+=item

    return result

File: Javascript/BaseballGame.js
var baseballGame = function(opearations){
    let stack = []

    for(const operation of operations){
        if(operation === 'C'){
            stack.pop()
        }
        else if(operation === 'D'){
            stack.push(2*stack[stack.length-1])
        }
        else if(operation === '+'){
            stack.push(stack[stack.length-1] + stack[stack.length-2])
        }
        else{
            stack.push(Number(operation))
        }
    }
    let result = 0
    for(const item of stack){
        result+=item
    }

    return result
}